src/js/tabs/osd.js

```diff
@@ -23,7 +22,7 @@ SYM.METRE = 0xC;
 SYM.FEET = 0xF;
 SYM.KPH = 0x9E;
 SYM.MPH = 0x9D;
-SYM.SPEED = 0x70;
+SYM.SPEED = function() { return semver.gte(CONFIG.apiVersion, "1.42.0") ? 0x70 : '' };
```

Comment:
I'd just return `SYM.BLANK` in the second case here, and avoid the check for empty string below.

Reply:
I did it in this way to try to let a more accurate preview. We need to customize the symbol in a version when this symbol does not exist in the older font version. Usually this implies that the OSD in the firmware does not print nothing. Putting a blank can let a strange preview if the symbol is in the middle, for example the ESC temperature, and will add spaces to the left that will make the preview very different than what the goggles show in others situations like the speed. I know the user must verify it before flying but I think this a better "best effort" for the OSD at very little cost.

Reply:
Not sure I understand that right. If we are trying to massage the characters we use so that a message that we are printing in the format used for one version looks like the message for another version then this will be very hard to maintain, and should be avoided.
If we want to support historical versions of a message, and the message format (as opposed to the character set used to print that message) was changed, then we need to store the historical formats in addition to the latest format.

Reply:
No, we are not trying to maintain historical message formats. I'm trying to display always the latest message format (as we have always done) but replacing "incorrect/ugly" characters by the most correct one. If the character exist (but was not used for this version) I let it, the user can identify the string and must verify in the goggle how it looks in its version, but if the character does not exist or it has been replaced by an incorrect character I try to fix it or remove it.

Reply:
I think not printing certain characters is fine. They're being dropped after the message is formatted, so it's not a huge thing. And with the disclaimer on previews. Half-assing is okay

Reply:
In this case I'd probably return a blank for the character we cannot print, or maybe even a visible placeholder for it, like a `*`. Completely removing the character will just change the alignment with other lines, and make it look even messier.